src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+import java.time.LocalDateTime;
+
+public class Event extends Task {
+
+    static char sym = '\u23F1';
```

Comment:
Similar to the deadline class, I think the variable name should be changed for clarity